test/api.test.ts

```diff
@@ -330,31 +371,74 @@ test('Draw.deleteAll when in direct_select mode', async () => {
 
   await afterNextRender();
 
-  assert.equal(Draw.getMode(), 'simple_select',
-    'switches to simple_select mode');
-  assert.equal(Draw.getAll().features.length, 0,
-    'removes selected feature along with others');
+  assert.equal(
+    Draw.getMode(),
+    'simple_select',
+    'switches to simple_select mode'
+  );
+  assert.equal(
+    Draw.getAll().features.length,
+    0,
+    'removes selected feature along with others'
+  );
 });
 
 test('Draw.changeMode and Draw.getMode with no pre-existing feature', () => {
   const drawInstance = Draw.changeMode('draw_polygon');
   assert.equal(drawInstance, Draw, 'changeMode returns Draw instance');
   assert.equal(Draw.getMode(), 'draw_polygon', 'changed to draw_polygon');
   assert.equal(Draw.getAll().features.length, 1, 'one feature added');
-  assert.equal(Draw.getAll().features[0].geometry.type, 'Polygon', 'and it is a polygon');
-  assert.deepEqual(Draw.getAll().features[0].geometry.coordinates, [[null]], 'and it is empty');
+  assert.equal(
+    Draw.getAll().features[0].geometry.type,
+    'Polygon',
+    'and it is a polygon'
+  );
+
+  assert.deepEqual(
+    Draw.getAll().features[0].geometry.coordinates,
+    [[undefined]],
```

Comment:
This needs to be checked. I'm not quite sure why it changed from `null` to `undefined`.